Modules/Core/Common/include/itkPointSet.h

```diff
@@ -156,6 +164,9 @@ class ITK_TEMPLATE_EXPORT PointSet : public DataObject
   void
   SetPoints(PointsContainer *);
 
+  void
+  SetPointsUsingArray(PointsVectorContainer *);
+
```

Comment:
Do we need a `GetPointsArray` here?

Reply:
No, output from GetPoints() can be converted to 2D numpy array using itk.array_from_vector_container